build_library/build_image_util.sh

```diff
@@ -154,6 +154,15 @@ write_packages() {
     fi
 }
 
+extract_docs() {
+    local root_fs_dir="$1"
+
+    info "Extracting docs"
+    tar --create --bzip --file="${BUILD_DIR}/docs.tar" \
```

Comment:
except for the directory tweak and name change, best to always have a top level directory that matches the tarball name.